pgx-examples/custom_sql/src/lib.rs

```diff
@@ -67,12 +67,14 @@ extension_sql!(
 
 // `extension_sql_file` does the same as `extension_sql` but automatically sets the `name` to the
 // filename (not the full path).
-extension_sql_file!("../sql/single.sql", requires = ["single_raw"]);
+extension_sql_file!("../sql/single.sql", name = "single", requires = ["single_raw"]);
```

Comment:
I see you added a test for `extension_sql` for this feature, could make one of the examples above do it too?

Reply:
Ya, I will add that. Interestingly I was unable to get `extension_sql_file` to work from the tests in `pgx-tests`. I couldn't figure out why though.

Reply:
Hmm you're right, if I add

```rust

extension_sql_file!(
    concat!("../../../pgx-examples/custom_sql/sql/single.sql"),
    name = "example_file",
);
```

To the test file it breaks on 

```
[2022-01-27 10:55:38.814 PST] [18676] [61f2eaaa.48f4]: LOG:  statement: ROLLBACK

relation "extension_sql_dynamic" does not exist
Postgres location: parse_relation.c:1177
Rust location: <unknown>

', pgx-tests/src/framework.rs:120:21
```

Reply:
If I add that, actually, all the `extension_sql` blocks disappear too??? So odd.

Reply:
done